packages/app-store/basecamp3/lib/constants.ts

```diff
@@ -0,0 +1 @@
+export const userAgent = "Cal.com (support@cal.com)";
```

Comment:
Need this in every request to comply with their API policies [here](https://github.com/basecamp/bc3-api/tree/master#identifying-your-application)

Reply:
This could present an issue, as this will not be applicable for self-hosters. I think this requirement should be clearly pointed out in the section of the README where explains how to get environment variable values and shape it as a new environment variable to be very explicit of its importance and to get the correct value for self-hosters.